google/cloud/internal/backoff_policy.h

```diff
@@ -128,9 +128,13 @@ class ExponentialBackoffPolicy : public BackoffPolicy {
                            double scaling)
       : initial_delay_(std::chrono::duration_cast<std::chrono::microseconds>(
             initial_delay)),
-        current_delay_range_(2 * initial_delay_),
         maximum_delay_(std::chrono::duration_cast<std::chrono::microseconds>(
             maximum_delay)),
+        current_delay_start_(initial_delay_),
+        current_delay_end_(
+            (std::min)(std::chrono::duration_cast<std::chrono::microseconds>(
+                           scaling * initial_delay_),
```

Comment:
Nit: I'd suggest moving `scaling_` up in the initialization order, so that we can use it here (instead of the parameter that we're otherwise about to copy).

Reply:
Done